### PR TITLE
Dashboard requests mobile: center content

### PR DESCRIPTION
**Pull request by Pineirin**
Move action buttons to a dropdown.
closes: inveniosoftware/invenio-requests#264

Screenshot from 2022-09-01 10-16-50
Screenshot from 2022-09-01 10-17-01
Screenshot from 2022-09-01 10-21-19

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/requests_items/RequestsResultProps.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { timestampToRelativeTime } from "../../utils";
+
+export const requestsResultProps = (result, updateQueryState, currentQueryState) => {
```
Comment (NRodriguezcuellar) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/requests_items/RequestsResultProps.js:9]: Is it a conscious decision to not import it from communities?

File: invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html [hunk 2]
```diff
@@ -43,7 +43,7 @@ <h2 class="ui header">{{ request.title }}</h2>
       </div>
     {% endif %}
 
-    <div id="request-actions" class="flex align-items-start ml-auto mb-10">
```
Comment (Pineirin) [inline invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html:46]: Before:
Screenshot from 2022-09-05 14-48-57

After:
Screenshot from 2022-09-05 14-49-08

Comment (kpsherva) [reply invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/macros/request_header.html:46]: was this discussed with @jennur ?

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/requests.js [hunk 3]
```diff
@@ -46,10 +46,14 @@ import {
   Icon,
   Segment,
 } from "semantic-ui-react";
-import { RDMRecordSearchBarElement } from "../search/components";
+import { ComputerTabletRequestActions } from "@js/invenio_requests/request/actions/requestActions/ComputerTabletRequestActions";
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/requests.js:49]: this import does not exist anymore

**Review by kpsherva — APPROVED** [review]

**Pull request merged**